@xen-orchestra/web/src/pages/(site)/hosts.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="hosts" :class="{ mobile: uiStore.isMobile }">
+    <UiCard class="container">
+      <HostsTable :hosts :has-error="hasHostFetchError" />
```

Comment:
```suggestion
      <HostsTable :hosts :is-ready="areHostsReady" :has-error="hasHostFetchError" />
```
Why did you remove this ?